examples/build_qwen3_decode_expand_mixed_kernel/passes_dump/10_after_FlattenTileNdTo2D.py

```
@@ -0,0 +1,332 @@
+                wq_chunk_0: pl.Tensor[[256, 64], pl.BF16] = pl.tensor.slice(wq_0, [256, 64], [k0_7, q0_0])
+                _t6: pl.Tensor[[4, 256], pl.BF16] = pl.tensor.cast(normed_0, target_type=pl.BF16, mode='round')
+                _t7: pl.Tensor[[4, 64], pl.BF16] = pl.tensor.matmul(_t6, wq_chunk_0, a_trans=False, b_trans=False, c_matrix_nz=False)
+                q_acc_4: pl.Tensor[[4, 64], pl.FP32] = pl.tensor.add(q_acc_iter_2, _t7)
+                k0_6, q_acc_3, x_chunk_6 = pl.yield_(k0_7, q_acc_4, x_chunk_7)
+            _t8: pl.Tensor[[4, 64], pl.BF16] = pl.tensor.cast(q_acc_3, target_type=pl.BF16, mode='round')
+            q_proj_5: pl.Tensor[[16, 5120], pl.BF16] = pl.tensor.assemble(q_proj_iter_3_outer_l1, _t8, [b0_0, q0_0])
+            k0_iter_3_outer_l1_rv, kb_iter_3_outer_l1_rv, q_proj_iter_3_outer_l1_rv, x_chunk_iter_3_outer_l1_rv = pl.yield_(k0_6, kb_5, q_proj_5, x_chunk_6)
+        return k0_iter_3_outer_l1_rv, kb_iter_3_outer_l1_rv, q_proj_iter_3_outer_l1_rv, x_chunk_iter_3_outer_l1_rv
+    @pl.function(type=pl.FunctionType.AIV)
+    def qwen3_decode_layer_incore_2(self, b0_0: pl.Scalar[pl.INDEX], gamma_iter_1_outer_l0: pl.Tensor[[1, 256], pl.FP32], hidden_states_0: pl.Tensor[[16, 5120], pl.BF16], input_rms_weight_0: pl.Tensor[[1, 5120], pl.FP32], inv_rms_tile_0: pl.Tensor[[4, 1], pl.FP32], k0_iter_8_outer_l0: pl.Scalar[pl.INDEX], k_proj_iter_3_outer_l0: pl.Tensor[[16, 1024], pl.BF16], kb_iter_6_outer_l0: pl.Scalar[pl.INDEX], normed_iter_1_outer_l0: pl.Tensor[[4, 256], pl.FP32], ob_1_out: pl.Scalar[pl.INDEX], v_proj_iter_3_outer_l0: pl.Tensor[[16, 1024], pl.BF16], wk_0: pl.Tensor[[5120, 1024], pl.BF16], wv_0: pl.Tensor[[5120, 1024], pl.BF16], x_chunk_bf16_iter_1_outer_l0: pl.Tensor[[4, 256], pl.BF16], x_chunk_iter_8_outer_l0: pl.Tensor[[16, 256], pl.FP32]) -> tuple[pl.Tensor[[1, 256], pl.FP32], pl.Scalar[pl.INDEX], pl.Tensor[[16, 1024], pl.BF16], pl.Scalar[pl.INDEX], pl.Tensor[[4, 256], pl.FP32], pl.Tensor[[16, 1024], pl.BF16], pl.Tensor[[4, 256], pl.BF16], pl.Tensor[[16, 256], pl.FP32]]:
+        for ob_1_in, (gamma_iter_1_outer_l1, k0_iter_8_outer_l1, k_proj_iter_3_outer_l1, kb_iter_6_outer_l1, normed_iter_1_outer_l1, v_proj_iter_3_outer_l1, x_chunk_iter_8_outer_l1, x_chunk_bf16_iter_1_outer_l1) in pl.parallel(8, init_values=(gamma_iter_1_outer_l0, k0_iter_8_outer_l0, k_proj_iter_3_outer_l0, kb_iter_6_outer_l0, normed_iter_1_outer_l0, v_proj_iter_3_outer_l0, x_chunk_iter_8_outer_l0, x_chunk_bf16_iter_1_outer_l0)):
```

Ruff (0.15.6)

[warning] Loop control variable `kb_iter_6_outer_l1` not used within loop body

Rename unused `kb_iter_6_outer_l1` to `_kb_iter_6_outer_l1`

(B007)